src/main/java/org/fundacionjala/coding/german/SortInnerContent.java

```diff
@@ -0,0 +1,50 @@
+package org.fundacionjala.coding.german;
+
+import java.util.Arrays;
+
+
+/**
+ * Created by German on 8/29/2017.
+ */
+public class SortInnerContent {
+    /**
+     * @param words String
+     * @return String  words in descending order and only inner
+     */
+    public String sortInnerConntent(String words) {
+        String delimiter = " ";
+        String[] temp = words.split(delimiter);
+        StringBuilder temBuilder = new StringBuilder();
```

Comment:
You can use a StringJoiner instead of StringBuilder to avoid appending whitespaces